apps/src/javalab/JavalabEditor.jsx

```diff
@@ -298,6 +299,11 @@ class JavalabEditor extends React.Component {
         renameFileError: this.duplicateFileError(newFilename)
```

Comment:
Looking at the two error messages, the `duplicateValidationFileError` message is also what we'd want if the student named a file after hidden support code. Mind handling that case here as well?